### PR TITLE
Consistent phase units

### PR DESCRIPTION
**Pull request by dieris**
Unit of 2\pi

### CODE REVIEW

**Comment by caryan** [comment]
@dieris let's talk tomorrow - I'm not sold on radians

**Comment by blakejohnson** [comment]
Seems like fraction of a circle would be better?

File: src/pulses.jl [hunk 1]
```diff
@@ -183,9 +183,9 @@ function MEAS(q::Qubit)
 end
 
 function flat_top_gaussian(chan)
-	return [Uθ(chan, chan.shape_params["riseFall"], chan.shape_params["amp"], chan.shape_params["phase"], "gaussOn"),
-	Uθ(chan, chan.shape_params["length"], chan.shape_params["amp"], chan.shape_params["phase"], "constant"),
-	Uθ(chan, chan.shape_params["riseFall"], chan.shape_params["amp"], chan.shape_params["phase"], "gaussOff")]
+	return [Uθ(chan, chan.shape_params["riseFall"], chan.shape_params["amp"], chan.shape_params["phase"]/2π, "gaussOn"),
```
Comment (dieris) [inline src/pulses.jl]: this is to make it compatible with the definition in pyQGL

File: src/APS2.jl [hunk 2]
```diff
@@ -139,7 +139,7 @@ function create_wf_instrs!(instr_lib, wfs, pulses)
 	for p in pulses
 		wf = p.amp * QGL.waveform(p, DAC_CLOCK)
 		if !USE_PHASE_OFFSET_INSTRUCTION
-			wf *= exp(1im * p.phase)
+			wf *= exp(1im * 2π * p.phase)
```
Comment (dieris) [inline src/APS2.jl:142]: this is so that the standard unit is in fraction of a circle

**Pull request merged**